charts/gateway-helm/templates/infra-manager-rbac.yaml

```diff
@@ -1,3 +1,34 @@
+{{ if .Values.config.envoyGateway.provider.kubernetes }}
+{{ $kube := .Values.config.envoyGateway.provider.kubernetes }}
+{{ if and (not $kube.watch) ($kube.deploy) (eq $kube.deploy.type "GatewayNamespace") }}
```

Comment:
I think we may need another controls for this, other wise it won't with following:
- install EG with default values with helm
- update configmap with kubectl

Reply:
true, we might think how to add this rbac after installing